x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/user/AuthenticateResponse.java

```diff
@@ -41,7 +41,7 @@ public void writeTo(StreamOutput out) throws IOException {
     @Override
     public void readFrom(StreamInput in) throws IOException {
         super.readFrom(in);
-        if (in.getVersion().before(Version.V_6_6_0)) {
+        if (in.getVersion().before(Version.V_7_0_0)) {
```

Comment:
Why is this changed in this PR? This seems unrelated to me.

Reply:
removing